frontend/app/routes/public/renew/$id/terms-and-conditions.tsx

```diff
@@ -25,6 +25,7 @@ import { transformFlattenedError } from '~/utils/zod-utils.server';
 
 enum CheckboxValue {
   Yes = 'yes',
+  No = 'no',
```

Comment:
Nitpick: is `No` needed? I don't see it being used anywhere.

Reply:
TS/eslint complains about:
`76:25  error  Unnecessary conditional, both sides of the expression are literal values  @typescript-eslint/no-unnecessary-condition`

There might be another way tho. lemme check

Reply:
`.valueOf()` works, but I hate it for some reason

Reply:
No preference either way so I'm resolving.